### PR TITLE
updated docs for oracle-node v1.0.5 release

### LINKED ISSUE #64: Razor go documentation to be updated for v1.0.5
- [ ] Update vote command with docker run...
 - [x] Mention in the docs about risks of supplying a password and tell the user how to delete the command from history
 - [x] Mention backupNode flag info in vote command.
 - [x] SetConfig command needs to be updated

Few of these tasks where created in oracle-node repo here, https://github.com/razor-network/oracle-node/issues/942

### PR DESCRIPTION
**Pull request by Shrikant1212**
Fixes #64

### CODE REVIEW

**Review by yohanelly95 — APPROVED** [review]

**Comment by Yashk767** [comment]
Merge it only after `v1.0.5` (Public release) is released.
It's raised right now in order to help internal validators use updated commands for `v1.0.5-beta`(internal prod release).
@yohanelly95  @rajkharvar

File: docs/stake/testnet.md [hunk 1]
```diff
@@ -78,9 +78,10 @@ There are a set of parameters that are configurable. These include:
 - Gas Price: The value of gas price if you want to set manually. If you don't provide any value or simply keep it to 1, the razor client will automatically calculate the optimum gas price and send it.
 - Log Level: Normally debug logs are not logged into the log file. But if you want you can set `logLevel` to `debug` and fetch the debug logs.
 - Gas Limit: The value with which the gas limit will be multiplied while sending every transaction.
+- RPC Timeout: This is the threshold number of seconds after which any contract and client calls will time out.
```
Comment (Yashk767) [inline docs/stake/testnet.md:81]: can you change this to 
```
RPC Timeout: Number of seconds after which any contract and client calls will time out if it's not responding
```

**Review by Yashk767 — CHANGES_REQUESTED** [review]

**Comment by Yashk767** [comment]
https://deploy-preview-76--razor-docs-v2.netlify.app/docs/stake/mainnet#software-dependencies
Here the docker image url needs to be updated with v1.0.5 docker image url which we would get once it is released.

Just reminding, you can merge it after this change is done. @yohanelly95 @rajkharvar

**Review by SkandaBhat — APPROVED** [review]

**Review by yohanelly95 — APPROVED** [review]

File: docs/stake/testnet.md [hunk 2]
```diff
@@ -45,9 +45,9 @@ It is recommended to run a **Razor Node** using **Docker**. This is because you
 
 Docker: You can find more information about installing docker [here](https://docs.docker.com/engine/install/).
 
-Razor-Go(github): You can download the Razor-go:v1.0.0-mainnet from [here](https://github.com/razor-network/oracle-node/releases/tag/v1.0.0-mainnet).
+Razor-Go(github): You can download the Razor-go:v1.0.5-alpha from [here](https://github.com/razor-network/oracle-node/releases/tag/v1.0.5-alpha-patch2).
```
[the remark this answers is not in the document]
Comment (SkandaBhat) [reply docs/stake/testnet.md:48]: This is correct.

**Pull request merged**